baybe/acquisition.py

```diff
@@ -1,18 +1,39 @@
 """Adapter for making BoTorch's acquisition functions work with other models."""
-
+from functools import partial
 from inspect import signature
 from typing import Any, Callable, List, Optional, Type
 
 import gpytorch.distributions
 from attr import define
-from botorch.acquisition import AcquisitionFunction
+from botorch.acquisition import (
+    AcquisitionFunction,
+    ExpectedImprovement,
+    PosteriorMean,
+    ProbabilityOfImprovement,
+    UpperConfidenceBound,
+    qExpectedImprovement,
+    qProbabilityOfImprovement,
+    qUpperConfidenceBound,
+)
 from botorch.models.gpytorch import Model
 from botorch.posteriors import Posterior
 from botorch.posteriors.gpytorch import GPyTorchPosterior
-from torch import Tensor, cat, squeeze
+from torch import Tensor
 
 from baybe.surrogates.base import Surrogate
 
+acquisition_function_mapping = {
```

Comment:
@Scienfitz Here I renamed `mapping` to `acquisition_function_mapping`.

Reply:
ty